backon/src/sleep.rs

```diff
@@ -78,16 +80,18 @@ impl Sleeper for TokioSleeper {
     }
 }
 
-/// The default implementation of `Sleeper` utilizes `gloo_timers::future::sleep`.
-#[cfg(all(target_arch = "wasm32", feature = "gloo-timers-sleep"))]
+/// Sleeper implementation using `futures-timer` crate.
+///
+/// See the [`futures-timer` crate](https://docs.rs/futures-timer/latest/) for details.
+#[cfg(feature = "futures-timer-sleep")]
 #[derive(Clone, Copy, Debug, Default)]
-pub struct GlooTimersSleep;
+pub struct FuturesTimerSleeper;
```

Comment:
Please note that `GlooTimersSleep` is a publicly exposed struct. Modifying it would result in a breaking change, lead to the release of `backon 2.0`, which we would prefer to avoid.